### PR TITLE
*/*: missing dependencies

### PR DESCRIPTION
**Pull request by thesamesam**

### CODE REVIEW

File: dev-db/libdbi/libdbi-0.9.0.ebuild [hunk 1]
```diff
@@ -16,8 +16,7 @@ SLOT=0
 
 DOCS=( AUTHORS ChangeLog README README.osx TODO )
 
-RDEPEND=""
-DEPEND="${RDEPEND}
+BDEPEND="
 	>=sys-apps/sed-4
```
Comment (SoapGentoo) [inline dev-db/libdbi/libdbi-0.9.0.ebuild:20]: remove

File: dev-libs/libvarlink/libvarlink-16.ebuild [hunk 2]
```diff
@@ -14,8 +14,7 @@ SLOT="0"
 KEYWORDS="~amd64"
 IUSE="test"
 RESTRICT="!test? ( test )"
-RDEPEND=""
-DEPEND="${RDEPEND}"
+
 BDEPEND="
 	>=dev-util/meson-0.47.0
```
Comment (SoapGentoo) [inline dev-libs/libvarlink/libvarlink-16.ebuild:19]: remove, should come from the eclass

File: dev-ruby/nokogiri/nokogiri-1.11.1-r1.ebuild [hunk 3]
```diff
@@ -25,10 +25,12 @@ IUSE=""
 RDEPEND="${RDEPEND}
 	>=dev-libs/libxml2-2.9.10:=
 	>=dev-libs/libxslt-1.1.34
+	sys-libs/zlib
 	virtual/libiconv"
 DEPEND="${DEPEND}
 	>=dev-libs/libxml2-2.9.10
 	>=dev-libs/libxslt-1.1.34
+	sys-libs/zlib
```
Comment (SoapGentoo) [inline dev-ruby/nokogiri/nokogiri-1.11.1-r1.ebuild:33]: remove and fix the RDEPEND include

File: sci-libs/parmetis/parmetis-4.0.3-r1.ebuild [hunk 4]
```diff
@@ -22,16 +22,14 @@ KEYWORDS="amd64 arm64 x86 ~amd64-linux ~x86-linux"
...
 	!<sci-libs/metis-5"
 
 pkg_setup() {
 	if use openmp; then
-		if [[ $(tc-getCC)$ == *gcc* ]] && ! tc-has-openmp; then
-			ewarn "You are using gcc but openmp is not available"
-			die "Need an OpenMP capable compiler"
-		fi
+		tc-check-openmp
 	fi
```
Comment (SoapGentoo) [inline sci-libs/parmetis/parmetis-4.0.3-r1.ebuild:33]: change the whole block to
`[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-ruby/nokogiri/nokogiri-1.11.1-r1.ebuild (hunk 3 above)
Comment (thesamesam) [reply dev-ruby/nokogiri/nokogiri-1.11.1-r1.ebuild:33]: I don't want to mess with the (strange) patterns like this which are in a lot of Ruby ebuilds

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-db/libdbi/libdbi-0.9.0.ebuild [hunk 5]
```diff
@@ -8,32 +8,34 @@ inherit autotools
 DESCRIPTION="A database-independent abstraction layer in C"
 SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
 HOMEPAGE="http://libdbi.sourceforge.net/"
-LICENSE="LGPL-2.1"
 
+LICENSE="LGPL-2.1"
+SLOT="0"
 IUSE="doc static-libs"
 KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~m68k ~mips ppc ppc64 ~s390 sparc x86"
```
Comment (SoapGentoo) [inline dev-db/libdbi/libdbi-0.9.0.ebuild:14]: still wrong

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

**Review by SoapGentoo — APPROVED** [review]

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**